portable-atomic-util/tests/arc.rs

```diff
@@ -257,8 +257,10 @@ mod alloc_tests {
         assert_eq!(unsafe { &*ptr }, "foo");
         assert_eq!(arc, arc2);
 
-        // TODO: CoerceUnsized is needed to cast to Arc<dyn ..>
-        // let arc: Arc<dyn Display> = Arc::new(123);
+        #[cfg(portable_atomic_unstable_coerce_unsized)]
+        let arc: Arc<dyn Display> = Arc::new(123);
+        // TODO: This is a workaround in case CoerceUnsized is not available - remove once this is no longer needed
```

Comment:
one more place to update this TODO comment, with updated rewording:

```suggestion
        // TODO: This is a workaround in case CoerceUnsized is not available - remove this once it is no longer needed
```